templates/base.html

```diff
@@ -36,12 +36,7 @@
 
 <body class="antialiased w-full " x-data="{ menu: false }">
 
-{% comment %} URL-dependent navbar {% endcomment %}
-<div hx-get="/core/check_url_nav/"
-     hx-trigger="load"
-     hx-swap="outerHTML"
->
-</div>
+{% include 'partials/nav.html' %}
```

Comment:
It sort of looked like there might have been some reason to load this via htmx... but not sure what that was.

Need to include it in order to have the current route, so we can highlight the active link.

Also, I haven't removed the `check_url_nav` route... so maybe I should do that too :)

Reply:
> It sort of looked like there might have been some reason to load this via htmx... but not sure what that was.

There might have been some other use in mind at the time.

> Need to include it in order to have the current route, so we can highlight the active link.

Sounds good!

> Also, I haven't removed the `check_url_nav` route... so maybe I should do that too :)

A quick search shows no other uses, so possibly yes!

Reply:
Ok, I removed it